lib/builder.js

```diff
@@ -178,17 +179,24 @@ var buildTrees = function (bundles, config) {
 }
 
 var buildTree = function (tree, destination, config) {
-  if (destination.indexOf('systemjs/plugin') != -1) {
+  if (destination.indexOf('systemjs/plugin') != -1 ||
+    destination.indexOf('bower_components') != -1) {
```

Comment:
bower_components should not be ignored here. We want to bundle them together with the app code.